bot/check-build.sh

```diff
@@ -0,0 +1,300 @@
+    
+    found_line_with_failed=0
+    echo "${PLAY_RECAP}" | grep "failed=" > /dev/null
+    ec=$?
+    [[ ${VERBOSE} -ne 0 ]] && echo "FAILED=.ec=${ec}"
+    [[ ${ec} -eq 0 ]] && found_line_with_failed=0 || found_line_with_failed=1
+    [[ ${VERBOSE} -ne 0 ]] && echo "found_line_with_failed=${found_line_with_failed}"
+    
+    failed_eq_zero=0
+    echo "${PLAY_RECAP}" | grep "failed=0" > /dev/null
+    ec=$?
+    [[ ${VERBOSE} -ne 0 ]] && echo "FAILED=0.ec=${ec}"
+    [[ ${ec} -eq 0 ]] && failed_eq_zero=0 || failed_eq_zero=1
+    [[ ${VERBOSE} -ne 0 ]] && echo "failed_eq_zero=${failed_eq_zero}"
+fi
+
+found_tarballs=0
+tarballs=$(ls eessi-${eessi_version}-compat-linux-${eessi_arch}-*.tar.gz 2>&1)
+ec=$?
+[[ ${VERBOSE} -ne 0 ]] && echo "TARBALLS.ec=${ec}"
+if [[ ${ec} -eq 0 ]]; then
+    found_tarballs=0
+else
+    found_tarballs=1
+fi
+[[ ${VERBOSE} -ne 0 ]] && echo "found_tarballs=${found_tarballs}"
+
+# construct and write complete PR comment
+comment_template="<details>__SUMMARY_FMT__<dl>__DETAILS_FMT____ARTEFACTS_FMT__</dl></details>"
+comment_summary_fmt="<summary>__SUMMARY__ _(click triangle for details)_</summary>"
+comment_details_fmt="<dt>_Details_</dt><dd>__DETAILS_LIST__</dd>"
+comment_success_item_fmt=":white_check_mark: __ITEM__"
+comment_failure_item_fmt=":x: __ITEM__"
+comment_artefacts_fmt="<dt>_Artefacts_</dt><dd>__ARTEFACTS_LIST__</dd>"
+comment_artefact_details_fmt="<details>__ARTEFACT_SUMMARY____ARTEFACT_DETAILS__</details>"
+
+function print_br_item() {
+    format="${1}"
+    item="${2}"
+    echo -n "${format//__ITEM__/${item}}<br/>"
+}
+
+function print_br_item2() {
+    format="${1}"
+    item="${2}"
+    item2="${3}"
+    format1="${format//__ITEM__/${item}}"
+    echo -n "${format1//__ITEM2__/${item2}}<br/>"
+}
+
+function print_code_item() {
+    format="${1}"
+    item="${2}"
+    echo -n "<code>${format//__ITEM__/${item}}</code>"
+}
+
+function print_dd_item() {
+    format="${1}"
+    item="${2}"
+    echo -n "<dd>${format//__ITEM__/${item}}</dd>"
+}
+
+function print_list_item() {
+    format="${1}"
+    item="${2}"
+    echo -n "<li>${format//__ITEM__/${item}}</li>"
+}
+
+function print_pre_item() {
+    format="${1}"
+    item="${2}"
+    echo -n "<pre>${format//__ITEM__/${item}}</pre>"
+}
+
+function success() {
+    format="${comment_success_item_fmt}"
+    item="$1"
+    print_br_item "${format}" "${item}"
+}
+
+function failure() {
+    format="${comment_failure_item_fmt}"
+    item="$1"
+    print_br_item "${format}" "${item}"
+}
+
+function add_detail() {
+    actual=${1}
+    expected=${2}
+    success_msg="${3}"
+    failure_msg="${4}"
+    if [[ ${actual} -eq ${expected} ]]; then
+        success "${success_msg}"
+    else
+        failure "${failure_msg}"
+    fi
+}
+
+if [[ ${failed_eq_zero} -eq 0 ]] && [[ ${found_tarballs} -eq 0 ]]; then
+    status="SUCCESS"
+    summary=":grin: SUCCESS"
+else
+    status="FAILURE"
+    summary=":cry: FAILURE"
+fi
+
+# TODO adjust format to what NESSI bot uses
```

Comment:
Not sure what needs to happen here, but this mentions NESSI? Can it be removed, or is there still something that needs to be changed?